openverse_catalog/dags/database/image_expiration_workflow.py

```diff
@@ -21,7 +21,7 @@
     default_args={
         **DAG_DEFAULT_ARGS,
         "retry_delay": timedelta(seconds=15),
-        "execution_timeout": None,
+        "execution_timeout": timedelta(seconds=0),  # Just let these tasks run
```

Comment:
Possible clarification:
```suggestion
        "execution_timeout": timedelta(seconds=0),  # Let these tasks run with no timeout
```